master/buildbot/steps/slave.py

```diff
@@ -19,6 +19,7 @@
 
 # pylint: disable=wildcard-import
 # pylint: disable=unused-wildcard-import
+from buildbot.steps.worker import WorkerBuildStep
```

Comment:
I think you should add the noqa on that error line

switching the import will lead to import order error